components/desktop/gst-plugins-base1/Makefile

```diff
@@ -0,0 +1,81 @@
+#
```

Comment:
Please, move them to components/multimedia to be consistent with other gstreamer* components.